Tiltfile

```diff
@@ -416,7 +418,10 @@ def enable_providers():
 
 def get_providers():
     user_enable_providers = settings.get("enable_providers", [])
-    return {k: "" for k in user_enable_providers + always_enable_providers}.keys()
+    providers = set(user_enable_providers) | set(default_enable_providers)
+    if not settings.get("enable_core_provider", True):
+        return all_providers - set([core_provider_name])
```

Comment:
Should all_providers be providers here and in the line below?

(or the variable in l.421 be called all_providers)

Reply:
Oh, good point, my fault due to multiple refactoring.